src/applications/disability-benefits/all-claims/components/Authorization.jsx

```diff
@@ -0,0 +1,500 @@
+    scrollTo('topScrollElement');
+    waitForRenderThenFocus('va-alert h3');
+  };
+
+  const handlers = {
+    onSubmit: event => {
+      // This prevents this nested form submit event from passing to the
+      // outer form and causing a page advance
+      event.stopPropagation();
+    },
+    onAnchorClick: () => {
+      const checkbox = $('va-checkbox');
+      scrollTo(checkbox);
+      waitForRenderThenFocus('input', checkbox.shadowRoot);
+    },
+    onChange: event => {
+      const { checked } = event.target;
+      setFormData({ ...data, patient4142Acknowledgement: checked });
+
+      if (checked && hasError) {
+        setHasError(false);
+      }
+    },
+    onGoForward: () => {
+      // Validation ONLY happens on form submission attempt
+      if (data.patient4142Acknowledgement) {
+        setHasError(false);
+        goForward(data);
+      } else {
+        // Show error and move focus ONLY when Continue is clicked without checkbox
+        setHasError(true);
+        focusOnAlert();
+      }
+    },
+  };
+
+  const focusSection = (_, element) => {
+    const focusedElement = document.querySelector(element);
+
+    focusElement(element);
+
+    if (focusedElement.getAttribute('open') === 'false') {
+      focusedElement.setAttribute('open', 'true');
+    }
+  };
+
+  const privacyModalButton = (
+    <va-button
+      onClick={toggle4142PrivacyModal}
+      secondary
+      text="Review Privacy Act Statement"
+    />
+  );
+
+  return (
+    <>
+      {hasError && (
+        <AuthorizationAlert
+          hasError={hasError}
+          onAnchorClick={handlers.onAnchorClick}
+        />
+      )}
+      <h3>Authorize the release of non-VA medical records to VA</h3>
+      <p>
+        Only provide this authorization if you want The Department of Veterans
+        Affairs (VA) to obtain non-VA medical records on your behalf. If you’ve
+        already provided these records or intend to get them yourself, there’s
+        no need to fill out this authorization. Doing so will lengthen your
+        claim processing time.
+      </p>
+      <va-accordion>
+        <va-accordion-item
+          header="1. Expiration and how to cancel authorization"
+          id="section-one"
+          level="4"
+          open
+        >
+          <p className="vads-u-margin-top--0">
+            This authorization will automatically expire in 12 months from the
+            date you authorize the form.
+          </p>
+          <p>
+            Providing this authorization is voluntary, but if you don’t
+            authorize it, or if you revoke it before we receive necessary
+            information, it could prevent an accurate or timely decision on your
+            claim, and could result in denial or loss of benefits.
+          </p>
+          <p>
+            You may revoke this authorization in writing at any time, except to
+            the extent a source of information has already relied on it to take
+            an action.
```

Comment:
A "source of information" has relied on the authorization to take action? I have no idea what that means.